apps/web/playwright/event-types.e2e.ts

```diff
@@ -257,19 +257,23 @@ testBothFutureAndLegacyRoutes.describe("Event Types tests", () => {
         expect(await linkElement.getAttribute("href")).toBe(testUrl);
       });
 
-      // TODO: This test is extremely flaky and has been failing a lot, blocking many PRs. Fix this.
-      // eslint-disable-next-line playwright/no-skipped-test
-      test.skip("Can remove location from multiple locations that are saved", async ({ page }) => {
+      test("Can remove location from multiple locations that are saved", async ({ page }) => {
         await gotoFirstEventType(page);
 
         // Add Attendee Phone Number location
         await selectAttendeePhoneNumber(page);
 
         // Add Cal Video location
+        // sometimes the dropdown from previous location is open at the same time, causing there to be two "Cal Video (Gloabl)"
+        // eslint-disable-next-line playwright/no-wait-for-timeout
+        await page.waitForTimeout(500);
         await addAnotherLocation(page, "Cal Video (Global)");
 
         await saveEventType(page);
         await page.waitForLoadState("networkidle");
+        // Form is not made dirty if the save button is still in loading state.
+        // eslint-disable-next-line playwright/no-wait-for-timeout
+        await page.waitForTimeout(1000);
```

Comment:
Maybe it has fixed the test failure, or maybe it hasn't. However, setting a timeout is not ideal, as we're guessing the time and don't know the actual duration. This is one reason to make a test flaky

Reply:
I couldn't find any other workaround here. waitForTimeout is being used in some other places for the same reason of automation being too fast

Reply:
Yes, but also that is contributing to flakiness. waitForTimeout is never a great solution - if there is a situation that requires an artificial delay it's probably better to fix the underlying components rather than hiding it - agree with @anikdhabal here.

Reply:
@emrysal I agree but both these timeouts are to let the transitions in the elements to play out, which wouldn't be a problem at human speed. Checkout https://github.com/calcom/cal.com/pull/16349#discussion_r1734142126 for more context. I tried a combination of different locators, but unfortunately this is the only thing that works. I have updated it to wait for toast to disappear, but its essentially doing the same thing.

Reply:
Or else I can change those elements to not have transitions when in e2e. Lmk what do you prefer.